python/lsst/ts/scheduler/driver/feature_scheduler.py

```diff
@@ -94,6 +105,10 @@ def __init__(self, models, raw_telemetry, parameters=None, log=None):
 
         self.script_configuration = dict()
 
+        self.observation_database_name = self.default_observation_database_name
+
+        self.schema_converter = SchemaConverter()
+
         super().__init__(models, raw_telemetry, parameters, log=log)
 
     def configure_scheduler(self, config=None):
```

Comment:
Please add a type definition for the parameters and a return type for the method. And review all other methods as well.